languages/php/src/SecretsClient.php

```diff
@@ -36,7 +36,7 @@ public function get_by_ids(array $secret_ids): \stdClass
         $project_get_by_ids_request->ids = $secret_ids;
         $project_get_by_ids_request->validate();
         $secrets_command = new SecretsCommand();
-        $secrets_command->get_by_ids = $project_get_by_ids_request->jsonSerialize();
+        $secrets_command->getByIds = $project_get_by_ids_request->jsonSerialize();
```

Comment:
:question: Was this causing issues?

Reply:
The PHP schema defines it's `getByIds`, so i technically fixing the fixing the PHP language client for the `$bitwarden_client->secrets->get_by_ids`, which before this change would just throw error and it works fine now  - also part of `example.php`.
If you want i can revert it and can fix it as separate issue.

Reply:
Thank you for the clarification, I see what's going on here now. 
I'm good with the change.